app/src/org/commcare/google/services/analytics/FirebaseAnalyticsUtil.java

```diff
@@ -203,6 +203,13 @@ public static void reportSyncResult(boolean result, String trigger, String syncM
         reportEvent(CCAnalyticsEvent.SYNC_ATTEMPT, b);
     }
 
+    public static void reportInAppUpdateResult(boolean result, String failureReason) {
+        Bundle bundle = new Bundle();
+        bundle.putInt(FirebaseAnalytics.Param.SUCCESS, result ? 1 : 0);
```

Comment:
it's better to use `FirebaseAnalytics.Param.VALUE` as the parameter here as Firebase [auto-aggregrates over that](https://firebase.google.com/docs/reference/android/com/google/firebase/analytics/FirebaseAnalytics.Param#VALUE).

Reply:
Nice, thanks for the suggestion.